packages/admin-server/src/app.ts

```diff
@@ -160,6 +162,21 @@ export const createApp = async (): Promise<{app: Koa, config: Config}> => {
     prefix: config.appPrefix
   });
 
+
+
+  // file proxy
+  // @ts-ignore
+  proxies.proxy.on('proxyReq', function(proxyReq, req, res, options) {
+    let cookies = cookie.parse(req.headers['cookie'] || '');
+    const accessToken = cookies.accessToken || '';
+    proxyReq.setHeader('Authorization', `Bearer ${accessToken}`);
+  });
+  app.use(proxies('/files', {
+    target: config.graphqlEndpoint.replace('/graphql', ''),
```

Comment:
`config.graphqlEndpoint` is used for client-side rather than server-side.

The value is my cluster is 
```
GRAPHQL_ENDPOINT: /api/graphql
```

I think we need to introduce new env (e.g. `GRAPHQL_SVC_URL`) 
```
GRAPHQL_SVC_URL: http://primehub-graphql/api/graphql
```

and inject it like
https://github.com/InfuseAI/primehub/blob/master/chart/templates/admission/webhook-deploy.yaml#L31-L32